clmm/theory/func_layer.py

```diff
@@ -619,22 +623,27 @@ def compute_reduced_tangential_shear(
         Source redshift model, with the following supported options:
 
             * `discrete` (default): all sources at one redshift (if `z_source` is a float) \
-                or known individual source galaxy redshifts (if `z_source` is an array and \
-                `r_proj` is a float);              
-            * `distribution` : sources follow a redshift distribution function. In this case the averaged lensing efficiency parameter(s) will be computed
-            and used to approximate the reduced tangential shear;
-            * `averaged lensing efficiency` : the source redshift distribution is described by the averaged lensing efficiency parameter(s). 
-            In this case these parameters will directly be used to approximate the reduced tangential shear;
-                
-    gt_equation: str, optional            
-        Expression for the compupuation of the approximated averaged reduced tangential shear. This is not used if `z_src_model`='discrete'. The supported options are:
-            
-            * `applegate14` (default): use the equation (6) in Weighing the Giants - III (Applegate et al. 2014; https://arxiv.org/abs/1208.0605).
-            * `schrabback18`: use the equation (12) in Cluster Mass Calibration at High Redshift (Schrabback et al. 2017; https://arxiv.org/abs/1611.03866).              
-            
+            or known individual source galaxy redshifts (if `z_source` is an array and \
+            `r_proj` is a float);
+            * `distribution` : sources follow a redshift distribution function. In this case \
+            the averaged lensing efficiency parameter(s) will be computed and used to \
+            approximate the reduced tangential shear;
+            * `averaged lensing efficiency` : the source redshift distribution is described by \
+            the averaged lensing efficiency parameter(s). In this case these parameters will \
+            directly be used to approximate the reduced tangential shear;
+
+    gt_equation: str, optional
+        Expression for the compupuation of the approximated averaged reduced tangential shear. \
+        This is not used if `z_src_model`='discrete'. The supported options are:
```

Comment:
Need to be `` `z_src_model` = 'discrete'`` instead of  `` `z_src_model`='discrete'`` for the documentation to compile.